README.md

```diff
@@ -16,42 +23,135 @@ Comprehensive and easy to use Material Design theme and control library for the
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+</Application>
+```
+* If you want to target Material Design 3, alter the `ResourceDictionary` line in the snippet above to use `MaterialDesign3.Defaults.xaml`.
+
+* Alter your `MainWindow.xaml`
+
+```xml
+<Window [...]
+  Style="{StaticResource MaterialDesignWindow}"
+  [...] >
+```
+
+
+## Building the source
+
+To build the project, following is required:
+* Microsoft Visual Studio 2022
+* WPF workload
+
+This repository also contains 3 different demo applications:
+* `MaterialDesignDemo` - Reference WPF app with Material Design 2 styling, this contains all controls and is a recommended tool when developing apps with this library
+* `MaterialDesign3Demo` - Reference WPF app with Material Design 3 styling, under development
+* `MahAppsDragablzDemo` - Demo app for combining with Dragablz and MahApps
+
+## Screenshots
+
+> [!WARNING]
+> The screenshots below are taken from the Material Design 2 demo app.
+> It is possible that these are out of date. Run the demo app from latest source to see the latest version.
+
+<details>
+  <summary>Show images</summary>
+
+  ![Screenshot of WPF Material Design 2 demo application home page](src/web/images/screen-home.png)
+  ![Buttons](src/web/images/screen-buttons.png)
+  ![Toggles](src/web/images/screen-toggles.png)
+  ![Fields](src/web/images/screen-fields.png)
+  ![ComboBoxes](src/web/images/screen-comboboxes.png)
+  ![Palette](src/web/images/screen-palette.png)
+  ![Color Tools](src/web/images/screen-colortools.png)
+  ![Pickers](src/web/images/screen-pickers.png)
+  ![Icons](src/web/images/screen-iconpack.png)
+  ![Cards](src/web/images/screen-cards.png)
+  ![Menus and Toolbars](src/web/images/screen-menutoolbar.png)
+  ![Progress Bars](src/web/images/screen-progress.png)
+  ![Dialogs](src/web/images/screen-dialogs.png)
+  ![Lists](src/web/images/screen-lists.png)
+  ![Tree View](src/web/images/screen-treeview.png)
+  ![Sliders](src/web/images/screen-sliders.png)
+  ![Chips](src/web/images/screen-chips.png)
+  ![Typography](src/web/images/screen-typography.png)
+  ![Group Box](src/web/images/screen-groupbox.png)
+  ![Snackbars](src/web/images/screen-snackbars.png)
+  ![Elevation](src/web/images/screen-elevation.png)
+</details>
+
+
+## More examples
+
+* [Keboo/MaterialDesign.Examples](https://github.com/Keboo/MaterialDesignInXaml.Examples)
+* [doobry](http://materialdesigninxaml.net/doobry)
+* [F1ix](http://materialdesigninxaml.net/f1ix)
+* [Motion List](https://github.com/MaterialDesignInXAML/MotionList)
+
+## Contributing
+
+Before contributing code read the [Contribution Guidelines](.github/CONTRIBUTING.md)
+* GitHub issues are for bugs and feature requests.
+* For questions, help and chat in general, please use the [GitHub discussion tab](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/discussions) or the [Discord server][discord-server-url].
```

Comment:
```suggestion
* For questions, help and chat in general, please use the [GitHub discussion tab](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/discussions) or the [Discord server](https://discord.keboo.dev).
```

Reply:
I have a Markdown variable on the bottom of the document because the link is used in multiple locations in the README.md. I'll replace the variable value with `https://discord.keboo.dev`.

Reply:
Ah I missed that, thanks